File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PlantsCreatePlantingsRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PlantsCreatePlantingsRequestItem {
    @JsonProperty("ActualDate")
    public String actualDate;
    @JsonProperty("LocationName")
    public String locationName;
    @JsonProperty("PatientLicenseNumber")
    public String patientLicenseNumber;
    @JsonProperty("PlantBatchName")
    public String plantBatchName;
    @JsonProperty("PlantBatchType")
    public String plantBatchType;
    @JsonProperty("PlantCount")
    public Integer plantCount;
    @JsonProperty("PlantLabel")
    public String plantLabel;
    @JsonProperty("StrainName")
    public String strainName;
    @JsonProperty("SublocationName")
    public String sublocationName;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PlantsCreatePlantingsRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PlantsCreatePlantingsRequestItem(
    @JsonProperty("actualDate")
    val actualDate: String? = null,
    @JsonProperty("locationName")
    val locationName: String? = null,
    @JsonProperty("patientLicenseNumber")
    val patientLicenseNumber: String? = null,
    @JsonProperty("plantBatchName")
    val plantBatchName: String? = null,
    @JsonProperty("plantBatchType")
    val plantBatchType: String? = null,
    @JsonProperty("plantCount")
    val plantCount: Int? = null,
    @JsonProperty("plantLabel")
    val plantLabel: String? = null,
    @JsonProperty("strainName")
    val strainName: String? = null,
    @JsonProperty("sublocationName")
    val sublocationName: String? = null
)
